tensorboard/plugins/pr_curve/summary.py

```diff
@@ -164,6 +164,98 @@ def op(
         description,
         collections)
 
+
+def streaming_op(tag, labels, predictions, num_thresholds=200, weights=None,
+                 metrics_collections=None, updates_collections=None,
+                 display_name=None, description=None):
+  """Computes a precision-recall curve summary across batches of data.
+
+  This function is similar to op() above, but can be used to compute the PR
+  curve across multiple batches of labels and predictions, in the same style
+  as the metrics found in tf.metrics.
+
+  This function creates multiple local variables for storing true positives,
+  true negative, etc. accumulated over each batch of data, and uses these local
+  variables for computing the final PR curve summary. These variables can be
+  updated with the returned update_op.
+
+  Args:
+    tag: A tag attached to the summary. Used by TensorBoard for organization.
+    labels: The ground truth values, a `Tensor` whose dimensions must match
+      `predictions`. Will be cast to `bool`.
+    predictions: A floating point `Tensor` of arbitrary shape and whose values
+      are in the range `[0, 1]`.
+    num_thresholds: The number of evenly spaced thresholds to generate for
+      computing the PR curve.
+    weights: Optional `Tensor` whose rank is either 0, or the same rank as
+      `labels`, and must be broadcastable to `labels` (i.e., all dimensions must
+      be either `1`, or the same as the corresponding `labels` dimension).
+    metrics_collections: An optional list of collections that `auc` should be
+      added to.
+    updates_collections: An optional list of collections that `update_op` should
+      be added to.
+    display_name: Optional name for this summary in TensorBoard, as a
+        constant `str`. Defaults to `name`.
+    description: Optional long-form description for this summary, as a
+        constant `str`. Markdown is supported. Defaults to empty.
+
+  Returns:
+    pr_curve: A string `Tensor` containing a single value: the
+      serialized PR curve Tensor summary. The summary contains a
+      float32 `Tensor` of dimension (6, num_thresholds). The first
+      dimension (of length 6) is of the order: true positives, false
+      positives, true negatives, false negatives, precision, recall.
+    update_op: An operation that updates the summary with the latest data.
+  """
+  thresholds = [i * 1.0 / float(num_thresholds - 1)
```

Comment:
Does `i / float(` ... suffice? Without the `* 1.0`?

Reply:
Yep. Done.